### PR TITLE
Add count_above and count_below features

### PR DESCRIPTION
**Pull request by danielazevedo**
I created two simple features, which I needed for my project. These features return the percentage of values in a time series that are above/below a certain threshold.

### CODE REVIEW

**Comment by MaxBenChrist** [comment]
Hi @danielazevedo, welcome to tsfresh.

Can you add some unit tests for those features?

File: tsfresh/feature_extraction/settings.py [hunk 1]
```diff
@@ -146,7 +146,9 @@ def __init__(self):
             "energy_ratio_by_chunks": [{"num_segments": 10, "segment_focus": i} for i in range(10)],
             "ratio_beyond_r_sigma": [{"r": x} for x in [0.5, 1, 1.5, 2, 2.5, 3, 5, 6, 7, 10]],
             "linear_trend_timewise": [{"attr": "pvalue"}, {"attr": "rvalue"}, {"attr": "intercept"},
-                                      {"attr": "slope"}, {"attr": "stderr"}]
+                                      {"attr": "slope"}, {"attr": "stderr"}],
+            "count_above": [{"t": t} for t in [-1, 0, 1]],
```
Comment (MaxBenChrist) [inline tsfresh/feature_extraction/settings.py]: I don't think we should give defaults for this feature, as the usefullness highly depends on the value range for your time series.

Comment (danielazevedo) [reply tsfresh/feature_extraction/settings.py]: Indeed, so you are suggesting I should remove them from the ComprehensiveFCParameters?

**Comment by danielazevedo** [comment]
Sure, I've added some unit test for the count_above and count_below features

File: tsfresh/feature_extraction/settings.py (hunk 1 above)
Comment (MaxBenChrist) [reply tsfresh/feature_extraction/settings.py]: Yes, I would remove it

File: tests/units/feature_extraction/test_feature_calculations.py [hunk 2]
```diff
@@ -1160,6 +1160,30 @@ def test_change_quantiles(self):
         res = change_quantiles(np.random.rand(10000) * 1000, 0.1, 0.2, False, 'mean')
         self.assertAlmostEqual(res, -0.9443846621365727)
 
+    def test_count_above(self):
```
Comment (MaxBenChrist) [inline tests/units/feature_extraction/test_feature_calculations.py]: looks good!

File: tsfresh/feature_extraction/settings.py (hunk 1 above)
Comment (danielazevedo) [reply tsfresh/feature_extraction/settings.py]: By removing the features from the ComprehensiveFCParameters, it throws an error during the tests execution. The _test_default_calculates_all_features_ test fails, as it checks if, by default, the ComprehensiveFCParameters object contains all the existing features.

Comment (nils-braun) [reply tsfresh/feature_extraction/settings.py]: If any, then I would go only with the 0 as default parameter

Comment (danielazevedo) [reply tsfresh/feature_extraction/settings.py]: I added the 0 as the default parameter

**Comment by nils-braun** [comment]
There are some very strange things going on with our tests... In principle the test runs through, so no problem here

**Pull request merged**